client/settings/payments-and-transactions-section/statement-preview/style.scss

```diff
@@ -0,0 +1,67 @@
+.statement-preview {
+	display: grid;
+	grid-template-columns: 2.5fr 1fr;
+	border: thin solid #dddddd;
+	padding: 12px 11px;
+	font-size: 11px;
+	line-height: 16px;
+	color: #949494;
+	border-radius: 3px 3px 0 0;
+	border-bottom: none;
+	box-shadow: 0px 2px 6px rgba( 0, 0, 0, 0.05 );
+	position: relative;
+
+
+	@media screen and (max-width: 609px) {
+		&:not(:last-child) {
+			margin-bottom: 20px;
+		}
+	}
+
+	@media screen and (min-width: 800px) and (max-width: 1109px) {
+		&:not(:last-child) {
+			margin-bottom: 20px;
+		}
+	}
+
+	&:after {
```

Comment:
Nice job on making the zigzag bottom :+1:

Reply:
Thanks!